code/modules/martial_arts/martial.dm

```diff
@@ -33,6 +33,8 @@
 	var/in_stance = FALSE
 	/// If the martial art allows parrying.
 	var/can_parry = FALSE
+	/// The priority of which martial art is picked from all the ones someone knows.
```

Comment:
can you specifies what the range of values mean i.e. 0 = lowest priority   and X... being the highest priority?